swingset/src/main/java/com/nqadmin/swingset/formatting/SSFormattedTextField.java

```diff
@@ -235,16 +272,18 @@ public void propertyChange(final PropertyChangeEvent _pce) {
 	/**
 	 * color for the field that has the focus
 	 */
-	// TODO Add setters/getters and allow developer to customize
-	private final java.awt.Color focusColor = new java.awt.Color(204, 255, 255);
+	private java.awt.Color focusBackgroundColor = new java.awt.Color(204, 255, 255);
 
 	/**
      * Common fields shared across SwingSet components
      */
     protected SSCommon ssCommon = new SSCommon(this);
 
-	private java.awt.Color standardColor = null;
-
+	/**
+	 * Used to store background color prior to change following focusGained event
+	 * so that the color can be restored upon focusLost.
+	 */
+    private java.awt.Color standardBackgroundColor = null;
 	
 	/**
 	 * Creates a new instance of SSFormattedTextField
```

Comment:
Not suggesting a change, maybe a TODO.

Looking at the jdk source code for `JFormattedTextField` I see that all the constructors go through one method, by judicious use of `this(...)`. I noticed since I was experimenting with having a property change listener and had to add it in each of the costructors.

Reply:
Agree. Added a TODO.

Reply:
Just to note it, property change listener customizations could probably go in customInit().